package.json

```diff
@@ -25,6 +25,9 @@
     "prepublish": "yarn build",
     "lint": "node_modules/.bin/eslint './src/**/*.js'"
   },
+  "engines": {
```

Comment:
I think you can remove this lines because it's not compatible with node > 13

ex : error react-alert@7.0.2: The engine "node" is incompatible with this module. Expected version "^13.10.1". Got "15.12.0"